File: P2N_Pet_BackEnd/P2N_Pet_API/P2N_Pet_API/Database/PetShopModels/Promotion.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace P2N_Pet_API.Database.PetShopModels
{
    public partial class Promotion
    {
        public ulong Id { get; set; }
        public string Title { get; set; }
        public string Image { get; set; }
        public DateTime? Fromdate { get; set; }
        public DateTime? Todate { get; set; }
        public int? Status { get; set; }
        public ulong? Createuser { get; set; }
        public DateTime? Createdate { get; set; }
        public ulong? Updateuser { get; set; }
        public DateTime? Updatedate { get; set; }
    }
}

File: P2N_Pet_FrontEnd/P2N-Pet/projects/admin/src/app/models/promotion.ts
export class Promotion{
    public Id: number;
    public Title: string;
    public Image: string;
    public FromDate: string;
    public ToDate: string;
    public StatusText: string;
    public CreateUserName: string;
    public CreateDate: string;
    public UpdateUserName: string;
    public UpdateDate: string;
}

export class PromotionCondition {
    Title: string;
    Status: string;
  
    constructor() {
      this.Title = '';
      this.Status = '0';
    }
  }